scripts/verify.sh

```diff
@@ -59,7 +59,7 @@ docker run \
     --rm \
     "gcr.io/cloud-marketplace-tools/k8s/dev" \
     -- \
-    /scripts/driver_internal.sh \
+    /scripts/verify \
```

Comment:
Having pairs of the likes of `install` and `install.sh` under `scripts` is a bit confusing. Did we decide to follow up and inline this into the Makefile?